docs/source/getting_started/woodwork_types.ipynb

```diff
@@ -0,0 +1,386 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b95b28c1",
+   "metadata": {},
+   "source": [
+    "# Woodwork Typing in Featuretools\n",
+    "\n",
+    "Featuretools relies on having consistent typing across the creation of EntitySets, Primitives, Features, and feature matrices. Previously, Featuretools used its own type system that contained objects called Variables. Now and moving forward, Featuretools will use an external data typing library for its typing: [Woodwork](https://woodwork.alteryx.com/en/stable/index.html).\n",
+    "\n",
+    "Understanding the Woodwork types that exist and how Featuretools uses Woodwork's type system will allow users to build EntitySets that best represent their data, understand the possible input and return types for Featuretools' Primitives, and understand what features will get generated from a given set of data and primitives.\n",
```

Comment:
Nitpick: This sentence seems a bit long. Maybe we can breakdown into bullet points?

Reply:
I like the idea of breaking it into bullets. I pretty much split it up as is, but let me know if it could use some more restructuring to be more clear:

> Understanding the Woodwork types that exist and how Featuretools uses Woodwork's type system will allow users to:
      - build EntitySets that best represent their data
      - understand the possible input and return types for Featuretools' Primitives
      - understand what features will get generated from a given set of data and primitives.

Reply:
That looks good to me